xml/schema/types/signalmasts-2-9-6.xsd

```diff
@@ -36,6 +24,7 @@
       <xs:sequence minOccurs="0" maxOccurs="unbounded">
         <xs:element name="turnoutsignalmast" type="TurnoutSignalMastType" minOccurs="0" maxOccurs="unbounded" />
         <xs:element name="dccsignalmast" type="DccSignalMastType" minOccurs="0" maxOccurs="unbounded" />
+        <xs:element name="matrixsignalmast" type="MatrixSignalMastType" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
This change needs to be in a 4-7-1 versioned schema, since it introduces a completely new element, not an element we failed to include in earlier schema, but generated panels with anyway.